runner/operator/access/v1_0_0/merge_fastqs/input_template.json.jinja2

```diff
@@ -0,0 +1,10 @@
+{
+   "fastq1": {{fastq1_files}},
```

Comment:
I'm not sure I've actually tried using `null` here, but I'm assuming that it works